src/libraries/System.Net.Http/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -3,8 +3,8 @@
     <type fullname="System.Net.Http.DiagnosticsHandler">
       <method signature="System.Boolean IsGloballyEnabled()" body="stub" value="false" feature="System.Net.Http.EnableActivityPropagation" featurevalue="false" />
     </type>
-    <type fullname="System.Net.Http.HttpClient">
-      <method signature="System.Boolean IsNativeHandlerEnabled()" body="stub" value="true" feature="System.Net.Http.UseNativeHttpHandler" featurevalue="true" />
+    <type fullname="System.Net.Http.HttpClientHandler">
+      <method signature="System.Boolean IsNativeHandlerEnabled()" body="stub" value="false" feature="System.Net.Http.UseNativeHttpHandler" featurevalue="false" />
```

Comment:
Would it make sense to substitute on both true and false values?

Reply:
From what I've tested, the current xml does substitute on true and false values.